File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.java
class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        int len = nums.length;

        for(int i=0; i < len; i++){
            if(start + i < len && nums[start + i] == target)
            return i;
            if(start - i >= 0 && nums[start - i] == target)
            return i;
        }
        return -1;
    }
}

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        le = len(nums)
        mini = float('inf')

        for i in range(start, le):
            if nums[i]==target:
                mini = min(mini, abs(i - start))
                break
        
        for i in range(start, -1, -1):
            if nums[i] == target:
                mini = min(mini, abs(start - i))
                break
        
        return mini